server/authz/policy.rego

```diff
@@ -906,6 +906,20 @@ allow {
   action == write
 }
 
+# Global admins and maintainers can read and write the EULA.
+allow {
+  object.type == "mdm_apple_eula"
+  subject.global_role == admin
+  action == [read, write, list][_]
+}
```

Comment:
I haven't seen tests for this role, did they already exist for the old object type? I'm assuming this would be for get/download/upload/delete EULA?

Reply:
added some new tests for this.